### DISCUSSION TITLE
Similar to recent post with a bit more context.

### DISCUSSION THREAD

**Discussion by herboren** [drewnoakes/metadata-extractor#604] [Q&A] [answered]
Is there anyways to identify the Hex Tag Offsets within the Image File Directories that correlate with the given filetype? Example offsets but not pertaining to cr2 and other formats:

0x0008
0x0009
0x0010

I see things like 
**Exif SubIFD - ISO Speed Ratings - 200** 

Returned from an image, but what I am really looking for are the offsets, similar to:

**0x0010 - ISO Speed Ratings - 200**

If I can identify the offsets, I can research Canons db for these offests to provide a better description of what they are and used for.

Again these are just examples. I wish to either help to improve the functionality through a fork by providing more information about the offsets, such as descriptions of each offset. Not contained value, but meaning to provide aid to understanding what each offset means.

**Comment by drewnoakes** [comment, marked as answer]
You can access tag IDs, which are numerical identifiers for tags. These are available via the API.

To be clear, they're not offsets within the file or directory as such. They're just numerical identifiers. However if your goal is to look up those tags in external documentation, then the tag ID is definitely the thing you want.
